Add 'all' to vague terms in no-vague-titles

ismail-syed opened the pull request:
I think we should also flag `all` as a vague term. Noticed this while reviewing https://github.com/Shopify/sewing-kit/pull/755.

ismail-syed committed:
update yarn.lock

ismail-syed committed:
Add 'all' as a vague term

ismail-syed committed:
update changelog

ismail-syed commented on yarn.lock:
```diff
@@ -1569,7 +1569,7 @@ eslint-plugin-react@7.7.0:
 
 "eslint-plugin-shopify@file:./.":
-  version "22.1.0"
+  version "23.0.0"
```
`yarn install` on a fresh clone of master updates this. Our lock file is our of sync.

cartogram reviewed (commented):
Changes look great! I love this in theory, but what if we have a prop called `onAllImagesUploaded` and then in our test we have `decribe('onAllImageUploaded')?

ismail-syed committed:
Support variable names with all

ismail-syed commented:
@cartogram, good catch! I've updated the regex to only match cases where `all` is surrounded by spaces or used before a period, to help determine that he it's being used in a sentence. This prevents flagging `onAllImagesUploaded`.  I've added a test with the `onAllImagesUploaded` to verify as well. Any other cases you can think of?

cartogram reviewed (approved):
@ismail-syed brilliant! This LGTM!
